denoise/unsupervised-wiener/main.py

```diff
@@ -0,0 +1,35 @@
+from scipy.signal import convolve2d as conv2
+import numpy as np
+import matplotlib.pyplot as plt
+from unspvd_wiener import unsupervised_wiener
+from skimage import color
+from PIL import Image
+
+
+def main():
+    """Test script for unsupervised_wiener
+    """
```

Comment:
Breaking up the code blocks by sections is always a good idea to improve readability